lib/uktest/include/uk/test.h

```diff
@@ -200,10 +200,8 @@
 
 /* Custom print method. */
 #define uk_test_printf(fmt, ...)					\
-	printf("    "							\
-	       LVLC_TESTNAME ":"					\
-	       UK_ANSI_MOD_RESET "\t"					\
-	       fmt, ##__VA_ARGS__)
+	_uk_printk(KLVL_INFO, NULL, NULL,	\
```

Comment:
maybe keep some of the indention whitespaces for readability purposes. The library name could and should be kept, I think `__LIBNAME__`. Preferably you should also use `__NULL` instead of `NULL` to be on the safe side. `NULL` is provided by a libc, `__NULL` is provided by Unikraft types.

Reply:
I don't really understand. The libname and basename should be kept. The line number should return an error. I understood the `__NULL` part.

Reply:
Sorry for my confusing reply 😉 , it has many options for your preference in case you have a strong preference or point for something.